lib/buildpackutil.py

```diff
@@ -265,38 +275,29 @@ def _checkout_from_git_rootfs(directory, mx_version):
     )
 
 
-def fix_mono_config_and_get_env(dot_local_location, mono_lib_dir):
+def _get_env_with_monolib(mono_lib_dir):
     env = dict(os.environ)
-    env['LD_LIBRARY_PATH'] = mono_lib_dir
-
-    if not os.path.isfile(os.path.join(mono_lib_dir, 'libgdiplus.so')):
+    env['LD_LIBRARY_PATH'] = mono_lib_dir + '/lib'
```

Comment:
`os.path.join` might be preferred for this one and the next one, instead of using `+`